### ISSUE TITLE
What's the status of this project?

### ISSUE THREAD

**Issue by RMHogervorst** [dfalster/RZotero#4]
Hi dfalster,

What is the status of this project? How does it compare with RefManager?
If you have any next steps I would like to help.

Kind regards,
Roel

**Comment by dfalster** [comment]
Hi @RMHogervorst, It's sitting dormant, waiting for attention. thanks for stopping by! Since starting RefManageR has appeared, that looks quite nice and has some of the features I was hoping to create (e.g. looking up DOIs). There's also some overlap in the `ReadZotero` function. Main thing that is missing in both is a `WriteZotero` function. What i would ultimately like to do to is be able to clean up my Zotero library (see wishlist in README). So what RZotero would ideally become is an R interface to the [Zotero API](https://www.zotero.org/support/dev/web_api/v3/start).

**Comment by RMHogervorst** [comment]
So I guess a unified transformation from api to f.i. data frame and the reverse is the goal? I glanced at the documentation of zotero and as I understood it modifying a json object would be the way to go. See writing to api at zotero

**Comment by dfalster** [comment]
Yes, thought I wouldn't go to dataframe, rather a list. Typical workflow might be 

1. Read tem (or items) from Zotero
2. Modify item
3. Push modified items back to Zotero, replacing / modifying original
